### PR TITLE
International home page updates from design QA and cta is from cms

### PR DESCRIPTION
**Pull request by jessgilbert**
screencapture-international-trade-great-8012-international-2019-04-11-10_24_18
^^ Hero button disappears when no link or text are added in the cms, otherwise still appears as normal

### CODE REVIEW

File: core/templates/core/landing_page.html [hunk 1]
```diff
@@ -46,43 +46,42 @@ <h2 class="highlight heading-medium background-active-blue white-text">{{ page.t
...
-                  </div>
-                </div>
-                <button class="button button-blue" type="submit">Continue</button>
-              </form>
-            {% else %}
-              <div class="button-container">
-                <a href="{{ page.tariffs_link }}" class="button button-blue button-arrow-small">Continue</a>
-              </div>
-            {% endif %}
+				{% if features.INTERNATIONAL_TARIFFS_COUNTRY_SELECT_ON %}
```
Comment (richtier) [inline core/templates/core/landing_page.html]: funky indent - looks like tabs and spaces mixed

File: core/templates/core/landing_page.html [hunk 2]
```diff
@@ -46,43 +46,42 @@ <h2 class="highlight heading-medium background-active-blue white-text">{{ page.t
...
+				  <form class="country-selector-dialog" action="{{ page.tariffs_link }}" method="GET">
```
Comment (richtier) [inline core/templates/core/landing_page.html]: what url will `tariffs_link` be? is that view not CSRF protected?

File: requirements.txt [hunk 3]
```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements.txt requirements.in
+#    __main__.py compile requirements.in
```
Comment (Kenneth-Siu) [inline requirements.txt]: Oh noes

File: requirements_test.txt [hunk 4]
```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile
 # To update, run:
 #
-#    pip-compile --output-file requirements_test.txt requirements_test.in
+#    __main__.py compile requirements_test.in
```
Comment (Kenneth-Siu) [inline requirements_test.txt]: This too

File: core/sass/pages/_landing-page.scss [hunk 5]
```diff
@@ -113,12 +113,22 @@
...
+      }
+    }
+
     .tariffs-content {
       padding: 0;
     }
   }
 
-  @media (max-width: 1150px) {
+  @media (max-width: 900px) {
```
Comment (Kenneth-Siu) [inline core/sass/pages/_landing-page.scss]: Why is it 900px and not, say, 961px (our breakpoint for xl screens)?

File: core/sass/pages/_landing-page.scss [hunk 6]
```diff
@@ -113,12 +113,22 @@
...
   @media (max-width: 640px) {
+    .hero-with-cta{
+      .container{
+        .hero-text{
+          .button-arrow-small{
+            white-space: normal;
+          }
+        }
+      }
+    }
```
Comment (Kenneth-Siu) [inline core/sass/pages/_landing-page.scss]: This bit is probably easier to read if you just have them all in the same line.

`.hero-with-cta .container .hero-text .button-arrow-small {`

Alternatively, do you definitely need all of them in the selector?

Comment (jessgilbert) [reply core/sass/pages/_landing-page.scss]: To override the normal button css of no whitespace, as the hero cta has quite long text it needs to go on to two lines to work on mobile

**Pull request merged**